src/experimental/tools/VideoThumbnailLoader/prepare_source_buffer.ts

```diff
@@ -66,6 +69,26 @@ export default function prepareSourceBuffer(
         resetMediaElement(videoElement, objectURL);
       });
     }
+    if (isManagedMediaSource && "disableRemotePlayback" in videoElement) {
+      const disableRemotePlaybackPreviousValue = videoElement.disableRemotePlayback;
+      cleanUpSignal.register(() => {
+        /**
+         * Restore the disableRemotePlayback attribute to the previous value
+         * in order to leave the <video> element in the same state has it was set
+         * by the application before calling the RxPlayer.
+         */
+        if (disableRemotePlaybackPreviousValue !== undefined) {
```

Comment:
I thought that if `videoElement.disableRemotePlayback` was defined, it was always a boolean. We already checked that `"disableRemotePlayback" in videoElement`, so how come `disableRemotePlaybackPreviousValue` can now be set to `undefined` for TypeScript?

Reply:
Yes, done.